src/platforms/javascript/common/sourcemaps/tools/webpack.mdx

```diff
@@ -15,6 +15,16 @@ npm install --save-dev @sentry/webpack-plugin
 yarn add --dev @sentry/webpack-plugin
 ```
 
+<PlatformSection supported={["javascript.nextjs"]}>
+<Note>
+
+If you‘ve installed the Next.js SDK, you don‘t have to install the plugin.
```

Comment:
This note will only appear in the Next.js guide, thus we could as well omit the plugin installation above.

Otherwise this can confuse users who will install the plugin and then either see the note and regret the previous action or ignore the note altogether.